setup.py

```diff
@@ -351,7 +351,12 @@ def get_extensions():
 
     ffmpeg_exe = distutils.spawn.find_executable('ffmpeg')
     has_ffmpeg = ffmpeg_exe is not None
-    if sys.platform != 'linux':
+    # Building torchvision with ffmpeg on MacOS or with Python 3.9 causes
```

Comment:
Should we add a FIXME or a TODO comment? It makes it easy to find out all outstanding issues with a simple `git grep command`

Reply:
Yeah, makes sense.